blackberggroup/VA-OHIL
# Feat/jls 7206944561/iex updates

## What type of PR is this? (check all applicable)


## Description
Fix "About" menu expand more/less overrides: chevron down/up should not display
Adjust iEX menu from "iEX 2023" to "iEX"
Adjust iEX links paths to point from iex-2023 to point to new page at iex-2024

## Related Tickets & Documents
- Monday Task #7206944561
- https://blackberg-group.monday.com/boards/1931719392/pulses/7206944561

## QA Instructions, Screenshots, Recordings
Confirm "About" menu does not display chevrons on hover/expand/collapse
Confirm iEX menu is "iEX" and no longer "iEX 2023" and links to new iEX 2024 page /hil/views/iex-2024
Confirm iEX link in the page content (bottom row, 3rd panel of divisions) goes to new iEX 2024 page /hil/views/iex-2024

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_


## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized 

## Changed Files in this Pull Request ##
- _includes/header-sub.html
 - _includes/header.html
 - _includes/home/divisions.html
 - _site/assets-3-5/css/styles.css
 - _site/assets/css/styles.css
 - _site/assets/sass/vha/components/navigation/_index.scss
 - _site/index.html
 - assets/css/styles.css
 - assets/sass/vha/components/navigation/_index.scss
